alpha/scripts/utils/userDisableKmcAccess.php

```diff
@@ -0,0 +1,67 @@
+<?php
+require_once (dirname(__FILE__) . '/../bootstrap.php');
+
+if ($argc < 2)
+{
+	echo "\n This Script revert kuser_id 'isAdmin' and 'loginEnabled' = 'login_data_id' to false\n * Input: {kuser_id} = PK or path/to/list.ext\n\n";
+	die (" Missing required parameters: php " . $argv[0] . " {kuser_id} [realrun / dryrun]\n\n");
```

Comment:
Specify that first parameter can be file name

Reply:
Added, will open a new PR for current server version